frontend/src/services/apiService.ts

```diff
@@ -1,5 +1,12 @@
 import { ofetch } from "ofetch"
 import { createFetch } from "@vueuse/core"
+import type { Router } from "vue-router"
+
+let routerInstance: Router | null = null
+
+export const initializeApiService = (router: Router) => {
```

Comment:
this feels like the api service should be a composable and use `useRouter` to get the router. But as the places using it are not all composables (yet) I think this solution works as well :)